packages/@aws-cdk/aws-s3/test/bucket.test.ts

```diff
@@ -495,6 +495,62 @@ nodeunitShim({
       test.done();
     },
 
+    'bucket with aws foundational security best practice enforce ssl'(test: Test) {
```

Comment:
```suggestion
    'can set enforceSsl'(test: Test) {
```

Reply:
Sorry for the many rounds :/ you can commit this one from theGitHub UI

Reply:
@NetaNir no worries! Ok I just commited that change.